incubator/nfs-client-provisioner/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
Use namespaced templates. Since Helm 2.7, this has been incorporated in `helm create`.

e. g. `{{- define "nfs-client-provisioner.name" -}}`